CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.0-nullsafety.0
```

Comment:
If there is no change in the public API, maybe this could be `1.1.0-nullsafety.0` ?

Reply:
Good point I'll fix that.